frontend/amundsen_application/config.py

```diff
@@ -83,6 +83,10 @@ class Config:
     PREVIEW_CLIENT_PASSWORD = os.getenv('PREVIEW_CLIENT_PASSWORD')  # type: Optional[str]
     PREVIEW_CLIENT_CERTIFICATE = os.getenv('PREVIEW_CLIENT_CERTIFICATE')  # type: Optional[str]
 
+    # Settings for Quality client
+    QUALITY_CLIENT_ENABLED = os.getenv('QUALITY_CLIENT_ENABLED') == 'true'  # type: bool
```

Comment:
this is probably unnecessary given that using presense of QUALITY_CLIENT is enough.